### DISCUSSION TITLE
Install on raspberry pi

### DISCUSSION THREAD

**Discussion by ludo453** [FreshRSS/FreshRSS#3618] [Q&A]
hello i can't find how to install it on raspberry pi. do you have a tutorial a help?

thank you

**Comment by Alkarex** [comment]
Hello,
I use a Raspberry Pi myself. Several options:
* use our [official Docker image](https://github.com/FreshRSS/FreshRSS/tree/edge/Docker) `freshrss/freshrss:arm` (updated documentation [coming soon](https://github.com/FreshRSS/FreshRSS/issues/3351)) 
* or see integrated systems such as [YunoHost](https://install-app.yunohost.org/?app=freshrss)
* or go for a classic installation using Apache + PHP following any guide for Raspbian (Raspberry Pi OS) or Debian or Ubuntu - they are all almost identical. See our documentation https://freshrss.github.io/FreshRSS/en/admins/03_Installation.html and https://freshrss.github.io/FreshRSS/en/admins/10_ServerConfig.html

**Reply by ludo453** [reply]
Hello,

Thank you for your answer.

Have a good day

**Reply by ludo453** [reply]
> Hello,
> I use a Raspberry Pi myself. Several options:
> 
> * use our [official Docker image](https://github.com/FreshRSS/FreshRSS/tree/edge/Docker) `freshrss/freshrss:arm` (updated documentation [coming soon](https://github.com/FreshRSS/FreshRSS/issues/3351))
> * or see integrated systems such as [YunoHost](https://install-app.yunohost.org/?app=freshrss)
> * or go for a classic installation using Apache + PHP following any guide for Raspbian (Raspberry Pi OS) or Debian or Ubuntu - they are all almost identical. See our documentation https://freshrss.github.io/FreshRSS/en/admins/03_Installation.html and https://freshrss.github.io/FreshRSS/en/admins/10_ServerConfig.html

Hi
how do you create a symbolic link to the ./p/ folder at the root of your web server?
thank you

**Reply by Alkarex** [reply]
That is not a good approach. You need to set the the root of your Web server to `./p/` but PHP still needs to be able to access `./p/../` which is to say the root folder of FreshRSS. See documentation links above, or that one in French https://github.com/FreshRSS/FreshRSS/blob/edge/README.fr.md#exemple-dinstallation-compl%C3%A8te-sur-linux-debianubuntu

**Reply by ludo453** [reply]
Hi

thank you I did not see that I will do it all again.

thank you

**Reply by ludo453** [reply]
hello I come back to you again is it possible to put it in https?

**Reply by Alkarex** [reply]
Yes. If you consider deploying with Docker, I suggest you use something like Traefik to have automatic HTTPS https://github.com/FreshRSS/FreshRSS/issues/2918#issuecomment-616668682
Otherwise, if you are with native Apache, see https://certbot.eff.org/lets-encrypt/debianbuster-apache

**Reply by ludo453** [reply]
Tank you

**Reply by the100rabh** [reply]
I also see freshrss/freshrss:latest-arm 

I am not confused as to which to use.

**Reply by Frenzie** [reply]
I believe that when in doubt, you should use the latest-* images.
